src/webgl/p5.RendererGL.Immediate.js

```diff
@@ -195,12 +195,12 @@ p5.RendererGL.prototype.endShape = function(
   this._processVertices(...arguments);
   this.isProcessingVertices = false;
   if (this._doFill) {
-    if (this.immediateMode.geometry.vertices.length > 1) {
+    if (this.immediateMode.geometry.vertices.length >= 3) {
```

Comment:
guessing this is because anything less than a triangle can't have a fill right? Just want to make sure there isn't some weird reason why this was originally set to 1

Reply:
Right! I noticed that in that example where we're drawing a bunch of lines, it was still trying to bind fill buffers. Lines don't have fills, and points are drawn using a separate `_drawPoints()` method, so I think this should be safe.